library-app/managers/interfaces/book_manager.go

```diff
@@ -0,0 +1,16 @@
+package interfaces
```

Comment:
In go interfaces are not usually defined in separate package, but are defined on the place where they are used.
See this article: https://www.alexedwards.net/blog/interfaces-explained
It explains the idea briefly.